magefiles/magefile.go

```diff
@@ -173,6 +177,13 @@ func Protoc() error {
 				return err
 			}
 		}
+
+		for _, file := range protoFiles {
+			if err := sh.RunV("protoc", ".", "paths=source_relative", "--go_out", ".", "--go_opt",
+				"paths=source_relative", file); err != nil {
+				return err
+			}
+		}
```

Comment:
If it is run in the container, would we need to update https://github.com/aquasecurity/trivy/blob/5e20c115fee9196943a3715c4f85bb3eac9d7d66/Dockerfile.protoc as well?

Reply:
See the alternate logic here https://github.com/aquasecurity/trivy/pull/7228/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR190-R195

Reply:
`mage:protoc` command builds an image and runs it. The dockerfile does not need to be updated.